src/blog/2019-census/index.md

```diff
@@ -0,0 +1,89 @@
+
+![A horizontal bar chart showing gender identity of Reactiflux members](./3-gender-identity.png)
+
+![A horizontal bar chart showing whether members of Reactiflux identify as transgender](./4-transgender.png)
+
+![A horizontal bar chart showing the sexual orientation of Reactiflux members](./5-sexual-orientation.png)
+
+![A horizontal bar chart showing the proportion of Reactiflux members who live with a disability](./6-disability.png)
+
+![A horizontal bar chart showing what religion Reactiflux members identify with](./7-religion.png)
+
+![A horizontal bar chart showing whether Reactiflux members feel their identity is represented within the community](./8-representation.png)
+
+![A horizontal bar chart showing the proportion of Reactiflux members who have felt marginalized or been harassed within the server](./9-bad-experiences.png)
+
+Reactiflux is predominently young (78% 18-34), white (57%), and male (89%).
+
+### Free response
+
+Bucketed into general type of response, we received:
+
+- 22 suggestions
+- 58 with positive feedback
+- 9 with neutral feedback
+- 8 with negative feedback
+- 20 who disliked the survey
+- 8 non-sequiturs
+
+The suggestions were about anything from new channels, new rules to enforce, new initiatives the submitter would enjoy, or more specific forms of "I don't like what you're doing, please do X instead".
+
+My personal favorite non-sequitors:
+
+> Vegan cheese really does not compare to the real deal
+
+> VALHALLAAAAAA
+
+Those who disliked the survey generally regarded it as either an overreach (along the lines of, "why do you want to know who I'm attracted to") or an encroachment of an ideology they opposed (along the lines of, "identity doesn't matter, let's focus on technology").
+
+Those with negative feedback, broadly, had had a bad experience with another member. We're glad to get word of these incidents, but our options for responding to these reports—weeks or months after the altercation—are limited. We're taking this as a signal that we need stronger mechanisms to report bad behavior within the server.
+
+We also received a small number of responses that expressed their frustrating with what they felt was a clique of Reactiflux insiders, both within the MVP/Mod/Admin group and regulars with no specific role. This is a concern of ours, but one that we feel we have to balance with being friendly: we want to avoid cliques that exclude others, but still allow people to be friends within the context of Reactiflux. Any group that has been bound by a single intent for multiple years—in this case, answering programming questions—is bound to develop some internal social ties and places of friction, but we want to keep that circle open.
+
+## Where we stand
+
+We're dismayed to see how heavily male-skewed the population is. While digging into other demographic surveys, though, we found these results are mirrored in other large communities.
```

Comment:
Sonic brought up that this reads as though we were shocked by it, but it wasn't really a surprise. That's a good point.

```suggestion
While not surprising, we're not happy with how heavily male-skewed the population is. While digging into other demographic surveys, though, we found these results are mirrored in other large communities.
```